MSProject/LibraryCollectionView/WBLibraryCollectionView.swift

```diff
@@ -13,70 +13,5 @@ class WBLibraryCollectionView: UIView, NibLoadable {
 
     @IBOutlet weak var libraryCollectionView: UICollectionView!
```

Comment:
Sorry for not noticing before and for nitpicking now, but this name for the collection view makes it kinda confusing when referencing it in the controller (`libraryCollectionView.libraryCollectionView`). Since the class it's called `WBLibraryCollectionView` already, maybe something like `bookCollection` would suit it best.